tests/modules/annotations/resources/test_matching_set.py

```diff
@@ -69,23 +63,25 @@ def test_annotation_matching_set(
     response = annot_utils.create_annotation(
         flask_app_client,
         researcher_1,
-        asset_guid,
-        enc_guid,
+        enc1_asset_guid,
+        enc1_guid,
         viewpoint=viewpoint,
     )
 
     annotation_guid = response.json['guid']
     annotation = Annotation.query.get(annotation_guid)
-    assert annotation.asset_guid == uuid.UUID(
-        test_clone_asset_group_data['asset_uuids'][0]
-    )
+    assert annotation.asset_guid == uuid.UUID(enc1_asset_guid)
+
     request.addfinalizer(annotation.delete)
     # must have this for matching
     annotation.content_guid = uuid.uuid4()
 
     # now we need a few other annots to see how they fair in matching_set creation
-    uuids = enc_utils.create_encounter(flask_app_client, researcher_1, request, test_root)
-    enc2_guid = uuids['encounters'][0]
+    enc2_uuids = enc_utils.create_encounter(
```

Comment:
Some name changes here to make it clear which was from the first and which from the second encounter. Previuosly the test was doing some strange things with asstes in the cloned AG which were in neither of teh crreated AGs.